### PR TITLE
Add do-while/while for CCFG, CAST

### PR DESCRIPTION
**Pull request by kriw**
- [x] CCFG::ActionNode::{While, DoWhile}
- [x] CAST::{new_while, new_do_while}
- [x] Unit tests for CCFG
- [x] Unit tests for CAST

### CODE REVIEW

**Comment by kriw** [comment]
Changes:
- Renamed
    - CASTConverter::ast -> cfg
    - CCFG::ast -> -> g
- Added a member `ast` to CASTConverter
- Splitted CASTConverter::to_c_ast_body into functions

File: src/backend/lang_c/c_ast.rs [hunk 1]
```diff
@@ -448,17 +478,28 @@ impl CAST {
...
+                        condition,
+                        while_body,
                         format_with_indent("}", indent))
             }
+            CASTNode::DoWhile => {
+                // Get the arguments -> while header/check condition, while body.
+                let args = self.get_args_ordered(node);
+                let condition = self.emit_c(&args[0], 0);
+                let while_body = self.emit_c(&args[1], indent + 1);
+                format!("{} {{\n{}}}\n{}({})",
```
Comment (HMPerson1) [inline src/backend/lang_c/c_ast.rs]: `do-while` ends with a semicolon:
```c
do {
  doStuff();
while (c);
```

**Comment by kriw** [comment]
This PR is ready to be merged if tests are passed.

File: src/backend/ctrl_flow_struct/graph_utils/test.rs [hunk 2]
```diff
@@ -1,217 +0,0 @@
-use super::*;
```
Comment (HMPerson1) [inline src/backend/ctrl_flow_struct/graph_utils/test.rs]: wait why did this whole file get deleted

**Pull request merged**